jenkins_pipelines/environments/common/pipeline.groovy

```diff
@@ -0,0 +1,86 @@
+def run(params) {
```

Comment:
As we moved to those groovy scripts, it might be interesting to reuse some pieces of code like methods.
For instance here, I see that the first two stages are identical.

Reply:
Absolutely right. I propose we evaluate it for a future PR (for now I wanted to avoid complicating the stuff for current pipeline developers, so we could stay as closer to the original as possible).